project_templates/latex_lsstdoc/templatekit.yaml

```
@@ -1,6 +1,6 @@
 name: "Change-controlled doc (lsstdoc LaTeX)"
 group: "Documents"
-dialog_title: "New lsstdoc LaTeX doc"
+dialog_title: "New Change Controlled  LaTeX doc"
```

```suggestion
dialog_title: "New lsstdoc LaTeX doc"
```

This needs to be 24 characters or less for Slack. Let's keep the original title.